assignments/27-02-25/oop/inheritance/Family.py

```diff
@@ -0,0 +1,36 @@
+class Grandfather:
+    def __init__(self, name):
+        self.__name = name
+
+    def __private_method(self):
+        print("This is a private method.")
+
+    def public_method(self):
+        print(f"The family member's name is {self.__name}")
+        self.__private_method()
+
+Grandfather = Grandfather("Ashraf", "5'7")
```

Comment:
It should be like this since the `Grandfather` constructor has only one property `name`:
`Grandfather = Grandfather("Ashraf") `

Reply:
The object name should be in lower case like this `grandfather`. Please change the line like this:
`grandfather = Grandfather("Ashraf", "5'7")`